packages/go_router/lib/src/redirection.dart

```diff
@@ -119,6 +124,27 @@ class RedirectionError extends Error implements UnsupportedError {
       ].join(' => ')}';
 }
 
+/// A configuration exception detected while processing redirects.
+class RedirectionException implements Exception {
```

Comment:
this should probably extend from GoError

Reply:
Exceptions aren't errors... but with all of the assert changes this class isn't used anymore, so I'll take it out.

Reply:
c6bbccd1261e9427a85aeb5f663c4f8df43be081